package.json

```diff
@@ -1,25 +1,41 @@
 {
   "name": "pliers-svg-sprite",
   "version": "0.1.0",
-  "description": "",
+  "description": "Pliers wrapper for svg-sprite",
   "author": "Jack Brewer",
-  "main": "pliers-svg-sprite.js",
+  "license": "BSD",
```

Comment:
License should be a valid SPDX license expression. 

I would suggest switching to `ISC`

http://spdx.org/licenses/